log/log.go

```diff
@@ -64,6 +60,44 @@ func init() {
 	baseLogger = logger{origLogger.Sugar()}
 }
 
+func getLogLevel() zapcore.Level {
+	levelStr := env.GetAsString("LOG_LEVEL", "info")
+
+	switch levelStr {
+	case "debug":
+		return zapcore.DebugLevel
+	case "info":
+		return zapcore.InfoLevel
+	case "warn":
+		return zapcore.WarnLevel
+	case "error":
+		return zapcore.ErrorLevel
+	default:
+		return zapcore.InfoLevel
+	}
+}
+
+func getEncoder() zapcore.Encoder {
+	encoderConf := zap.NewProductionEncoderConfig()
+
+	// Set RFC3339 timestamp encoding format
+	encoderConf.TimeKey = "timestamp"
+	encoderConf.EncodeTime = func(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
+		enc.AppendString(t.Format(time.RFC3339))
+	}
+	encoderConf.CallerKey = "source"
+
+	useConsoleEncoder, err := strconv.ParseBool(env.GetAsString("CONSOLE_LOGGER", "false"))
```

Comment:
maybe do this instead:
`useConsoleEncoder := strings.EqualFold(os.Getenv("CONSOLE_LOGGER"), "true")`